### ISSUE TITLE
Initial import takes 1 second

### ISSUE THREAD

**Issue by smarie** [smarie/python-mini-lambda#5]
```python
import time
start_time = time.time()
import mini_lambda
print("--- %s seconds ---" % (time.time() - start_time))
```

We can probably have all the goodies from math lib, etc. **not** imported by default but only on demand ? This requires some investigation

**Comment by smarie** [comment]
Thanks a lot @semiversus for the feedback. I am surprised that it is simply the numpy and pandas loading time. So is it the case that if we comment out the lines in `goodies.py` startup becomes fast ? If so, then I suggest that we put those lines in dedicated files that are not loaded in the global __init__, but that need to be explicitly loaded by users (`from mini_lambda.numpy import X` and `from mini_lambda.pandas import df`). Of course in this case we would remove the `try except` since failed loading is not a problem anymore. What do you think ?

**Comment by semiversus** [comment]
The times I measured are 0.35s with *numpy* and *pandas* and 0.05s without. Loading them explicitly seems to be the easiest way - I could imagine that other frameworks will be included too in the future.

BTW: I'm working on [broqer](https://github.com/semiversus/python-broqer) and also use operator overloading for some tricks. I also recognized thar for ``__len__``, ``__int__`` and others you're not able to return something else than the expected type. Saw your solution to introduce functions like ``Len`` and ``Int`` to be able to mimic the functionality. Will shamelessly use the idea for broqer ;-)
